coalib/bears/Bear.py

```diff
@@ -22,6 +24,34 @@
 from .meta import bearclass
 
 
+class db(pdb.Pdb):
+    def do_continue(self, arg):
```

Comment:
Shouldn't it be called somehow differently? Otherwise you override `do_c` / `do_continue` implicitly, and it will now always clear breakpoints, regardless of doing `q` or `c`.

Reply:
No it will not IMO thats how `cmd.Cmd()` is working, if we don't explicitly  which command do what then it will not interfere those commands which is defined in `super()`

Reply:
Actually `do_c` or `do_cont` will not but ya `do_continue` will, so making it `custom_quit`